<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWoSftpPermissionDialog</class>
 <widget class="QDialog" name="QWoSftpPermissionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File permission</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Path:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="path"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>User:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkURead">
       <property name="text">
        <string>readable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkUWrite">
       <property name="text">
        <string>writable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkUExecute">
       <property name="text">
        <string>execute</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Group:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkGRead">
       <property name="text">
        <string>readable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkGWrite">
       <property name="text">
        <string>writable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkGExecute">
       <property name="text">
        <string>execute</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Other:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkORead">
       <property name="text">
        <string>readable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkOWrite">
       <property name="text">
        <string>writable</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkOExecute">
       <property name="text">
        <string>execute</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QCheckBox" name="chkSubdirs">
       <property name="text">
        <string>Include Subdirectories</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Permission:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="permOld">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="permNew">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="tip">
     <property name="styleSheet">
      <string notr="true">color: rgb(165, 29, 45);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCommit">
       <property name="text">
        <string>Commit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
